Classes/Fusion/Helper/NodeInfoHelper.php

```diff
@@ -137,6 +137,7 @@ public function renderNodeWithMinimalPropertiesAndChildrenInformation(NodeInterf
 
         if ($controllerContext !== null) {
             $nodeInfo = array_merge($nodeInfo, $this->getUriInformation($node, $controllerContext));
+            $nodeInfo = array_merge($nodeInfo, $this->getBackendUriInformation($node, $controllerContext));
```

Comment:
We have to be careful here to create more data. I imagine this could affect performance for large uncollapsed document trees?

Can we test/verify that?